.github/workflows/ci.yml

```diff
@@ -271,4 +271,4 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Build BitGoJS Express Docker Image
-        run: podman build .
+        run: ./scripts/build-docker-express.sh
```

Comment:
Should we enforce that there is no diff in the dockerfile as well? That way devs are forced to commit changes to the dockerfile when adding or removing coins

Reply:
> Should we enforce that there is no diff in the dockerfile as well? 

yes! I'll add a new CI job specifically for that in the next PR.

Reply:
Here it is -> https://github.com/BitGo/BitGoJS/pull/7110